src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,13 @@
+package duke;
```

Comment:
Perhaps you might want to consider using this class help to handle more situations so that the Parser class does not have to handle each case individually